02/dive.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from typing import List  # Python 3.8 and earlier
+
+
+def calculate_moved_distance(movements: list) -> int:
+    """For a given set of movement inputs calculates the horizontal and vertical distance moved and returns the product of the two values
+
+    Parameteres:
+      movements (list): List of movement inputs - Each list entry, is a pair of [movement direction, movement value]
+
+    Returns
+      movement_product (int): Product of the horizontal and vertical distance moved
+    """
+
+    x = 0
+    y = 0
```

Comment:
Pedant point - I think it's really x and z, because depth for me is really 'z' -> I think I used 'horizontial_distance' and 'depth' as my variable names